models/category.js

```diff
@@ -9,8 +9,14 @@ const categorySchema = new Schema(
       required: true
     },
     categoryIcon: {
-      type: String,
-      required: true
+      path: {
+        type: String,
+        required: true
```

Comment:
ass validation message to all req fields
[true, 'This field cannot be empty.']

Reply:
okey, i have added